casbin/synced_enforcer.py

```diff
@@ -566,4 +566,7 @@ def remove_grouping_policies(self,rules):
     def remove_named_grouping_policies(self,ptype,rules):
         """ removes role inheritance rules from the current named policy."""
         with self._wl:
-            return self._e.remove_named_grouping_policies(ptype,rules)
\ No newline at end of file
+            return self._e.remove_named_grouping_policies(ptype,rules)
+
+    def build_incremental_role_links(self, op, ptype, rules):
+        self.get_model().build_incremental_role_links(self.get_role_manager(), op, "g", ptype, rules)
```

Comment:
According to the go version of casbin `build_incremental_role_links` should be a method of `Enforcer`.

I think `SyncedEnforcer` should then wrap `Enforcer.build_incremental_role_links` (in other versions of casbin `build_incremental_role_links` is not synchronized)

e.g.
```python
def build_incremental_role_links(self, op, ptype, rules):
    with self._wl:
        return self._e.build_incremental_role_links(op, ptype, rules)
```

@techoner do you agree?

Reply:
@abichinger  I agree with your suggestion, it should be so .

Reply:
@abichinger @techoner Should `SyncedEnforcer` inherit `Enforcer` then?

Reply:
> While I implemented the SyncedEnforcer I was unsure if inheritance is the right choice.
I decided against it, because SyncedEnforcer would automatically inherit newly added methods of Enforcer, which are not synchronized by default.

I would say it shouldn't inherit `Enforcer`.